target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -641,6 +642,7 @@ ramips_setup_macs()
 		wan_mac=$(mtd_get_mac_binary devdata 0x7)
 		;;
 	edimax,br-6478ac-v2|\
+	iptime,a104ns|\
```

Comment:
So, nothing on flash for the address?

Reply:
Unlike A1004ns, A104ns has just garbage value at 0x1fc40. How weird.